src/modules/endpoint/sagas/__tests__/loadEvents.spec.js

```diff
@@ -66,4 +68,14 @@ describe('loadEvents', () => {
 
     expect(await dataContainer.getEvents(city, language)).toBe(otherEvents)
   })
+
+  it('should fetch events if the stored JSON is malformatted', async () => {
+    const context = new DatabaseContext('augsburg', 'de')
+    const path = new DatabaseConnector().getContentPath('events', context)
+    await RNFetchBlob.fs.writeFile(path, `{ "i": { "am": "malformatted" } }`, 'utf-8')
+    const dataContainer = new DefaultDataContainer()
+    const events = await runSaga({}, loadEvents, city, language, dataContainer, false).toPromise()
```

Comment:
Why does this not throw an exception? I can not see that there is a new try-catch added in the DatabaseConnector.js.
Only for lastUpdate.

Reply:
Please have a look at the changes in `loadEvents`.